api/python/provisioner/hare.py

```diff
@@ -164,3 +171,42 @@ def ensure_cluster_is_started(tries=30, wait=10):
 def ensure_cluster_is_healthy(tries=1, wait=10):
     logger.info("Ensuring cluster is online and healthy")
     ensure(check_cluster_is_online, tries=tries, wait=wait)
+
+
+def r2_check_cluster_health_status() -> bool:
+    """
+    Wrapper over API of the hare which provides the capability to check
+    the health status of cluster.
+
+    Returns
+    -------
+    bool
+        True if the health status check of the cluster succeeds and
+        False otherwise
+
+    """
+    try:
+        res = cmd_run('cluster status', targets=LOCAL_MINION)
+    except Exception as e:
+        logger.debug(f'Some HA exception occured: {e}')
+        return False
+    else:
+        if not res:
```

Comment:
i feel it would be an unexpected case, please add some `logger.warning` here

Reply:
No more needed since we decided to use wrapper under R1 calls.